src/js/angular/aclmanagement/model.js

```diff
@@ -101,25 +162,77 @@ export class ACListModel {
     set aclRules(value) {
         this._aclRules = value;
     }
+
+    filterByScope(scope) {
```

Comment:
This `filterByScope` is kind of OK as a name given what it does, although you are technically not filtering but just getting some value from the map by key, but in the places where you use it, it sounds confusing. The missing jsdoc contributes to this confusion too.

Reply:
renamed and added documentation